doc/java_project.md

```diff
@@ -1,23 +0,0 @@
-# Java Project
```

Comment:
Let's leave this file. We can replace the contents with something like "Java Project support is planned for a future release". I'd rather not remove entirely because IIRC it's linked from a number of pages.

Reply:
Ditto for the other language-specific pages.